cfme/fixtures/ansible_fixtures.py

```diff
@@ -145,6 +145,34 @@ def ansible_catalog_item(appliance, ansible_repository):
     cat_item.delete_if_exists()
 
 
+@pytest.fixture(scope="function")
+def ansible_catalog_item_funcscope(appliance, ansible_repository):
+    collection = appliance.collections.catalog_items
+    cat_item = collection.create(
+        collection.ANSIBLE_PLAYBOOK,
+        fauxfactory.gen_alphanumeric(),
+        fauxfactory.gen_alphanumeric(),
+        display_in_catalog=True,
```

Comment:
Option:
It is good to add here a blank line to improve code readability

Reply:
Yes @dgaikwad